packages/svelte-table/package.json

```
@@ -20,37 +20,40 @@
     "svelte-table",
     "datagrid"
   ],
-  "type": "commonjs",
-  "module": "build/lib/index.esm.js",
-  "main": "build/lib/index.js",
-  "types": "build/lib/index.d.ts",
+  "type": "module",
+  "module": "build/index.js",
+  "main": "build/index.js",
+  "types": "build/index.d.ts",
   "exports": {
     ".": {
-      "types": "./build/lib/index.d.ts",
-      "import": "./build/lib/index.mjs",
-      "default": "./build/lib/index.js"
+      "types": "./build/index.d.ts",
+      "import": "./build/index.js",
+      "svelte": "./build/index.js"
```

Svelte should probably come first
```suggestion
      "svelte": "./build/index.js",
      "import": "./build/index.js"
```